libraries/Microsoft.Bot.Builder.AI.LUIS/LuisRecognizer.cs

```diff
@@ -31,7 +31,7 @@ public class LuisRecognizer : IRecognizer
         /// </summary>
         public const string LuisTraceLabel = "Luis Trace";
         private const string _metadataKey = "$instance";
-        private readonly LuisRuntimeAPI _runtime;
+        private readonly ILUISRuntimeClient _runtime;
```

Comment:
Casing. In many examples ("LuisApplication", "LuisTraceLabel", "LuisPredictionOptions") LUIS is lower case. In this case it's LUIS. 

Please choose one. :)

Reply:
This is from the underlying LUIS SDK--where they have changed the names.  I'm happy to go to all LUIS--that was my preference but did not seem to be the naming guidelines we were following in the V4 SDK.

Reply:
I'm going to leave this as Luis in all of SDK V4.